rebaser_test.go

```diff
@@ -70,33 +70,33 @@ func testRebaser(t *testing.T, when spec.G, it spec.S) {
 	when("#Rebase", func() {
 		when("app image and run image exist", func() {
```

Comment:
If we keep the input paramter to `Rebase`, I think we should add a test that sends in a different value for the destination and assert we did not change the given `appImage`.

Reply:
I don't quite follow. Would that be a different test than [this one](https://github.com/buildpacks/lifecycle/blob/50646839c80cc899193a8ed587ad59a20ec3f15a/rebaser_test.go#L357-L363)?

Reply:
That test is checking `fakeAppImage.SavedNames()` against `additionalNames, outputImageRef` but it isn't checking that the `previous-image` is NOT touched